docs/source/bp/network_segmentation.rst

```diff
@@ -0,0 +1,49 @@
+..
+  Created by: mike garcia
+  To: network segmentation
+
+.. |bp_title| replace:: Segment Networks Based on Sensitivity
+
+|bp_title|
+----------------------------------------------
+
+Network Segmentation is the practice of splitting a network into multiple sub-networks. These networks are usually designed around business needs, for example, having sub-networks for executives, finance, operations, and human resources or by keeping election functions separated from other county functions. Networks can also be separated by data sensitivity, keeping more sensitive election data separated from every day communications.
```

Comment:
Do we want to talk about something like segmenting users from different network areas? Or at the very least, how network segmentation may require users to re-authenticate in order to gain access to more sensitive areas of the enterprise?